docs/glossary/README.md

```diff
@@ -0,0 +1,287 @@
+
+Admins of a project can also create **dashboards that are visible to everyone in the project**.
+
+## Date alerts
+
+Date alerts in OpenProject are an Enterprise add-on and defined as a feature to generate automatic and customized notifications regarding a work package's due date or start date. You can find the date alerts feature in your notification center, symbolized by a little bell on the right upper side of your instance. [Read more about the date alerts feature in our user guide](https://www.openproject.org/docs/user-guide/notifications/notification-settings/#date-alerts-enterprise-add-on) or in [this article on deadline management with OpenProject](https://www.openproject.org/blog/deadline-management-date-alert/)
+
+## Enterprise add-on
+
+In OpenProject, some features are defined as an Enterprise add-on. This means, they are not part of the Community edition and therefore not free of charge. Enterprise add-ons are available as Cloud or on-premises versions. You can chose from the following plans to get access to all Enterprise add-ons: Basic, Professional, Premium and Corporate. [Read more about OpenProject plans and pricing](https://www.openproject.org/pricing/)
+
+## Effort (shirt size)
+
+## Excel synchronisation
+
+https://www.openproject.org/docs/system-admin-guide/integrations/excel-synchronization/
+
+## Forum
+
+A forum in OpenProject is defined as a module used to display forums and forum entries. The module has to be activated in the project settings and a forum has to be created in the forums tab in the project settings to be displayed in the side navigation. [Read more about forums in OpenProject](https://www.openproject.org/docs/user-guide/forums/)
+
+## File link
+
+## File storage
+
+## Filters
+
+Filters are essential for task and project management in OpenProject. You have several filtering options applicable to work package list. Filter options can be saved via *custom queries*. A filtered work packages list view (e.g. only open work packages) can be safed and therefore work as a *dashboard*. 
+
+## Forum
+
+In OpenProject, a forum is defined as a module used to display forums and forum entries. The module has to be activated in the project settings and a forum has to be created in the forums tab in the project settings to be displayed in the side navigation.
+
+## Gantt chart
+
+The Gantt chart in OpenProject displays the work packages in a timeline. You can collaboratively create and manage your project plan. Have your project timelines available for all team members and share up-to-date information with stakeholders. You can add start and finish dates and adapt it with drag and drop in the Gantt chart. Also, you can add dependencies, predecessor or follower within the Gantt chart.
+
+[Read more about how to activate and work with Gantt charts in OpenProject](https://www.openproject.org/docs/user-guide/gantt-chart/)
+
+![glossary-openproject-gantt-chart](glossary-openproject-gantt-chart.png)
```

Comment:
Please provide a human readable image description. (e.g. this is read aloud by screenreaders)
e.g.
`
![A gantt chart in OpenProject](glossary-openproject-gantt-chart.png)
`

Please make sure that the file size of the image is in our [recommended limits](https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#save-images). e.g. this file is 560KB

Reply:
Thank you, I didn't know the recommended limits yet.